synapse/lib/cell.py

```diff
@@ -1186,6 +1196,63 @@ def _getCellLock(self):
             mesg = f'Cannot start the {ctyp}, another process is already running it.'
             raise s_exc.FatalErr(mesg=mesg) from None
 
+    async def _onBootOptimize(self):
+
+        lmdbs = []
+        for (root, dirs, files) in os.walk(self.dirn):
+            for dirname in dirs:
+                if os.path.isfile(os.path.join(root, dirname, 'data.mdb')):
+                    lmdbs.append(os.path.join(root, dirname))
+
+        if not lmdbs:
+            return
+
+        logger.warning('Beginning onboot optimization (this could take a while)...')
+
+        size = len(lmdbs)
+
+        try:
+
+            for i, lmdbpath in enumerate(lmdbs):
+
+                logger.warning(f'... {i+1}/{size} {lmdbpath}')
+
+                with self.getTempDir() as backpath:
+
+                    s_t_backup.backup(lmdbpath, backpath)
```

Comment:
This path is one level too deep for what backup expects and ends up just copying the lmdb files rather than backing up them correctly, causing them to no longer be sparse.